kuma/static/js/wiki.js

```diff
@@ -191,11 +191,21 @@
         Syntax highlighting scripts
     */
     if($('article pre').length && ('querySelectorAll' in doc)) (function() {
-        var syntaxScript = doc.createElement('script');
-        syntaxScript.setAttribute('data-manual', '');
-        syntaxScript.async = 'true';
-        syntaxScript.src = mdn.staticPath + 'js/syntax-prism-min.js?build=' + mdn.build;
-        doc.body.appendChild(syntaxScript);
+        var bundle = 'syntax-prism';
+        if (window.mdn && mdn.assets.js && mdn.assets.js.hasOwnProperty(bundle)) {
```

Comment:
We assume the existence of `mdn` elsewhere, so we can probably do it here too by omitting the `window.mdn` check.